Urban-Fun/View Controllers/HostViewController.m

```diff
@@ -38,7 +39,47 @@ - (void)viewDidLoad {
         [self.view addGestureRecognizer:tapGestureRecognizer];
     }
 
+- (IBAction)didTapImage:(id)sender {
+    UIImagePickerController *imagePickerVC = [UIImagePickerController new];
+    imagePickerVC.delegate = self;
+    imagePickerVC.allowsEditing = YES;
+    
+
+   
+    
+    UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Upload Image"
```

Comment:
Here, instead of creating the `UIAlertController` we can potentially add support to the `HelperClass` to be able to receive callbacks (blocks) as a parameters when showing the alert controller, then you can reuse the code here and pass down the `handler` for Ok and Cancel actions.

Another potential solution here is to use `UIAlertControllerStyleActionSheet` as style to present 3 options, "Camera", "Library" or "Cancel" since this alert controller doesn't let the user to cancel the action

Reply:
implemented!